pkg/kubelet/dockershim/docker_container.go

```diff
@@ -458,6 +458,17 @@ func (ds *dockerService) ContainerStatus(_ context.Context, req *runtimeapi.Cont
 		Labels:      labels,
 		Annotations: annotations,
 		LogPath:     r.Config.Labels[containerLogPathLabelKey],
+		Resources: &runtimeapi.ContainerResources{
```

Comment:
We should not be adding support to dockershim, it's slated for complete removal next release.

Reply:
Dockershim is gone now. I've adapted (weakened) pod-resize-e2e tests to verify that resize was actuated by looking at cgroup values. Until containerd adds support for KEP 2273, the test will fail to verify e2e resize by looking at podStatus. I've coded this as a beta blocker in the pod resize e2e test so this cannot get accidentally overlooked.